data-analytics/iot-nirvana/app-engine/pom.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Add copyright header, inceptionYear, organization name and url.

Reply:
Done in commit 72fc217a2cbf413f3d3a082e17d523aa4b7595bc